config/2gig/ct32.xml

```diff
@@ -217,14 +221,14 @@
     </Value>
     <Value genre="config" index="11" label="Simple UI Mode" max="1" min="0" size="1" type="list" units="" value="1">
       <Help>
-        Simple UI Mode Enable/Disable
+        If the value is set to Disable then Normal Mode is enabled. If the value is set to Enable then Simple Mode is enabled.
       </Help>
       <Item label="Enable" value="0"/>
       <Item label="Disable" value="1"/>
     </Value>
     <Value genre="config" index="12" label="Multicast" max="1" min="0" size="1" type="byte" units="" value="0">
       <Help>
-        Multicasting Enable/Disable
+        If set to 0, multicast is disabled, if set to 1, will enable the multicast.
```

Comment:
You could potentially change this to a list (or bool)

Reply:
I thought of doing that, but didn't want to break anything. I'll go ahead and update it considering the state of all the others.